### PR TITLE
Feature_Friendly Locator

### PR DESCRIPTION
**Pull request by prashelke**
Friendly Locator related changes
Thank you for your contribution.
Before submitting this PR, please make sure:
- [x] PR description and commit message should describe the changes done in this PR
- [x] Verify the PR is point to correct branch i.e. Release or Beta branch if the code fix is for specific release , else point it to master
- [x] Latest Code from master or specific release branch is merged to your branch
- [x] No unwanted\commented\junk code is included
- [x] No new warning upon build solution
- [x] Existing unit test cases are passed
- [ ] New Unit tests are added for your development
- [ ] Sanity Tests are successfully executed for Existing Functionality
- [x] After creating pull request there should not be any conflicts
- [ ] Resolve codacy comments
- [ ] Builds and checks are passed before PR is sent for review
- [ ] Resolve code review comments

### CODE REVIEW

File: Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs [hunk 1]
```diff
@@ -3463,6 +3467,41 @@ public IWebElement LocateElementByLocators(ObservableList<ElementLocator> Locato
             return null;
         }
 
+        public IWebElement LocateElementByFriendlyLocators(ElementLocator locator, ObservableList<ElementLocator> FriendlyLocators, ElementInfo currentPOMElementInfo)
+        {
+            IWebElement friendlyEle = null;
```
Comment (Maheshkale447) [inline Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs:3472]: add try catch to handle the exception in the this function

File: Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs [hunk 2]
```diff
@@ -5452,13 +5516,171 @@ ObservableList<ElementLocator> IWindowExplorer.GetElementLocators(ElementInfo El
...
                 }
             }
             locatorsList = new ObservableList<ElementLocator>(locatorsList.Where(x => x.IsAutoLearned).ToList());
             return locatorsList;
         }
 
+        ObservableList<ElementLocator> IWindowExplorer.GetElementFriendlyLocators(ElementInfo ElementInfo)
+        {
+
+            ObservableList<ElementLocator> locatorsList = new ObservableList<ElementLocator>(); //new Platforms.PlatformsInfo.WebPlatform().GetLearningLocators();
```
Comment (Maheshkale447) [inline Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs:5538]: add exception handling to this function

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs (hunk 1 above)
Comment (prashelke) [reply Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs:3472]: Added

File: Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs (hunk 2 above)
Comment (prashelke) [reply Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs:5554]: Added

**Pull request merged**